src/Microsoft.Azure.WebJobs/TimeoutAttribute.cs

```diff
@@ -26,11 +26,30 @@ public sealed class TimeoutAttribute : Attribute
         public TimeoutAttribute(string timeout)
         {
             Timeout = TimeSpan.Parse(timeout, CultureInfo.InvariantCulture);
+            TimeoutGracePeriod = TimeSpan.FromSeconds(2);
         }
 
         /// <summary>
         /// Gets the timeout value.
         /// </summary>
         public TimeSpan Timeout { get; private set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether function invocations will timeout when
+        /// a <see cref="Timeout"/> is specified and a debugger is attached. False by default.
+        /// </summary>
+        public bool TimeoutWhileDebugging { get; set; }
```

Comment:
I'll take 'em!

---
In reply to: [77077864](https://github.com/Azure/azure-webjobs-sdk/pull/799#discussion_r77077864) [](ancestors = 77077864)